lib/IDE/SourceEntityWalker.cpp

```diff
@@ -177,13 +176,11 @@ bool SemaAnnotator::walkToDeclPre(Decl *D) {
       }
       return false;
     }
-  } else {
-    return true;
```

Comment:
There's still a few places where walkToDeclPost would be called but the pre wouldn't, ie. in the `ValueDecl`/`ImportDecl`/`IfConfig` branches. Should I add those into the post?

Reply:
Turns out `ValueDecl` was fine, it has a weird inverted bool there. The other two I added to Post.